Wangscape/logging/Logging.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "AppenderType.h"
+#include "Level.h"
+#include "Logger.h"
+
+#include <ostream>
+
+#define LOG_ERROR getLogger(logging::Level::Error)
+#define LOG_WARNING getLogger(logging::Level::Warning)
+#define LOG_DEBUG getLogger(logging::Level::Debug)
+#define LOG_INFO getLogger(logging::Level::Info)
```

Comment:
> Could these macros be replaced with functions or static (const?) instances/references?

Yes, of course. With my approach I've also assumed that we won't need the second logger. There are a few ways to do logging, as you most likely know, and beside static functions we could have also e.g. an `Logger` class instance and proper methods to use (like in Python: https://docs.python.org/3/library/logging.html). What do you think about that?

We could have:
* defines :x:
```cpp
LOG_DEBUG << "error no" << 1;
```
* static functions like in Qt
```cpp
logDebug() << "error no" << 2;
```
* static functions, but accepting `std::string`/`const char*`:
```cpp
logDebug("error no " + std::to_string(3));
```
* logger methods (either returning stream with `operator<<` or accepting `std::string`):
```cpp
auto logger = getLogger();
logger.logDebug("error no " + std::to_string(4));
```